zeppelin-server/pom.xml

```diff
@@ -275,6 +276,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.stefanbirkner</groupId>
+      <artifactId>system-rules</artifactId>
```

Comment:
What is the license of this dependency ? You may need to update the LICENCE file.

Reply:
You are right, I fixed it.